relay-event-normalization/src/normalize/processor.rs

```diff
@@ -1732,4 +1770,51 @@ mod tests {
         process_value(&mut reprocessed2, &mut processor, ProcessingState::root()).unwrap();
         assert_eq!(reprocessed, reprocessed2);
     }
+
+    #[test]
+    fn test_renormalize_transactions_is_idempotent() {
+        let json = r#"{
+  "event_id": "52df9022835246eeb317dbd739ccd059",
+  "type": "transaction",
+  "transaction": "test-transaction",
+  "start_timestamp": 1,
+  "timestamp": 2,
+  "contexts": {
+    "trace": {
+      "trace_id": "ff62a8b040f340bda5d830223def1d81",
+      "span_id": "bd429c44b67a3eb4"
+    }
+  }
+}"#;
+
+        let mut processed = Annotated::<Event>::from_json(json).unwrap();
+        let processor_config = NormalizeProcessorConfig::default();
+        let mut processor = NormalizeProcessor::new(processor_config.clone());
+        process_value(&mut processed, &mut processor, ProcessingState::root()).unwrap();
+        remove_event_received(&mut processed);
```

Comment:
`event.received` is always updated with the latest timestamp, so removing it from the events makes us compare them easily. This is not new functionality and is already the case, see:
https://github.com/getsentry/relay/blob/6c294c96a83fd4c2bc1967aaf3ed9a890febded9/relay-event-normalization/src/normalize/mod.rs#L2173
We can probably get rid of some tests when we unify more functionality.